pkg/healthcheck/healthcheck.go

```diff
@@ -1152,11 +1161,36 @@ func (hc *HealthChecker) allCategories() []category {
 						return &SkipError{Reason: "not run for non HA installs"}
 					},
 				},
+				{
+					description: "multiple replicas of control plane pods",
+					hintAnchor:  "l5d-control-plane-replicas",
+					warning:     true,
+					check: func(ctx context.Context) error {
+						if hc.isHA() {
+							return hc.controlPlaneComponentsAreHA()
+						}
+						return &SkipError{Reason: "not run for non HA installs"}
+					},
+				},
 			},
 		},
 	}
 }
 
+func (hc *HealthChecker) controlPlaneComponentsAreHA() error {
+	for _, component := range linkerdHAControlPlaneComponents {
+		conf, err := hc.kubeAPI.AppsV1().Deployments(hc.ControlPlaneNamespace).Get(component, metav1.GetOptions{})
+		if err != nil {
+			return err
+		}
+
+		if conf.Status.AvailableReplicas <= 1 {
```

Comment:
Just to verify my understanding here @adleong refered to that as the  

> the check for at least 1 available replica is still valuable

However this effectivelly checks that there is more than one avaiable replicas and that is what it shold be? Correct? Also, since this number can change as pods come up and cross the `minReadySeconds` threshold to be considered Available, would it be worth to make that a retryable check, much like the `"data plane proxies are ready"` check ?

Reply:
@zaharidichev yep, it makes sense to me to make it retryable. On it!